### PR TITLE
Additional order fields

### PR DESCRIPTION
**Pull request by legendofa**
This PR adds the VAT number and an optional payment authorization field.

### CODE REVIEW

File: src/order.rs [hunk 1]
```diff
@@ -41,6 +42,13 @@ pub struct Order {
     pub compensatable_order_amount: u64,
     /// UUID of payment information that the order should be processed with.
     pub payment_information_id: Uuid,
+    /// Optional payment authorization information.
+    /// This field is not queriable with GraphQL.
+    #[graphql(skip)]
+    pub payment_authorization: Option<PaymentAuthorization>,
```
Comment (nk-coding) [inline src/order.rs]: Iirc this should only be passed via the order context but not persisted in the db, right @KiLanger ?

**Review by nk-coding — CHANGES_REQUESTED** [review]

File: src/order.rs [hunk 2]
```diff
@@ -41,6 +42,13 @@ pub struct Order {
...
+    /// VAT number.
+    #[graphql(skip)]
```
Comment (lesser1) [inline src/order.rs:46]: Does this `skip` thing mean that the VAT number is not queryable? Why shouldn't it be?

**Review by lesser1 — COMMENTED** [review]
Just reviewed the code. However, that doesn't mean I understood everything -- I have little to no experience in Rust. I just tried to make sense of it.

File: src/order.rs (hunk 2 above)
Comment (nk-coding) [reply src/order.rs:46]: because as we discussed with Steffen, it should be put on the invoice instead

File: src/order.rs (hunk 1 above)
Comment (legendofa) [reply src/order.rs]: It is now on the order placement mutation

**Comment by legendofa** [comment]
@lesser1 This is now a breaking change for the frontend, as the `place_order` mutation signature changed

**Review by nk-coding — APPROVED** [review]

**Pull request merged**